bin/kibana

```diff
@@ -21,4 +21,4 @@ if [ ! -x "$NODE" ]; then
   exit 1
 fi
 
-exec "${NODE}" $NODE_OPTIONS "${DIR}/src/cli" ${@}
```

Comment:
I haven't been able to find a way to selectively add this warning, besides potentially determining if the --dev flag is provided.